src/components/hv-view/index.js

```diff
@@ -40,6 +40,53 @@ export default class HvView extends PureComponent<HvComponentProps> {
 
   props: HvComponentProps;
 
+  isHiddenElement = (attributes: Attribute[]): boolean => {
+    for (let j = 0; j < attributes.length; j += 1) {
+      if (attributes[j].name === 'hide' && attributes[j].value === 'true') {
+        return true;
+      }
+    }
+    return false;
+  };
+
+  canRenderElement = (element: Element): boolean => {
+    // ignore non rendering and hidden elements
+    return (
+      !!this.props.options.componentRegistry &&
+      !!element.namespaceURI &&
+      !!element.localName &&
+      !!this.props.options.componentRegistry[element.namespaceURI] &&
+      !!this.props.options.componentRegistry[element.namespaceURI][
+        element.localName
+      ] &&
+      // $FlowFixMe
+      !this.isHiddenElement(Array.from(element.attributes ?? []))
+    );
+  };
+
+  getElementsToRender = (): Element[] => {
+    // do not include non rendering elements while calculating index
+    // $FlowFixMe
+    const childNodes: Element[] = Array.from(
+      this.props.element.childNodes ?? [],
+    );
+    return childNodes.filter(element => this.canRenderElement(element));
+  };
+
+  getStickyElementIndices = (): number[] => {
+    const elements = this.getElementsToRender();
+    return elements.reduce((acc, element, index) => {
+      // $FlowFixMe
+      const attributes: Attribute[] = Array.from(element.attributes ?? []);
+      attributes.forEach(attribute => {
```

Comment:
Rather than iterating through attributes, can you use `element.getAttribute("sticky") === "true"`?